test/tests.py

```diff
@@ -1070,15 +1070,23 @@ def run_cc(self, cc_extraargs):
 
 if get_simics_major() == "6":
     all_tests.append(CTestCase(
-        ["1.2", "errors", "T_WREF"],
+        ["1.2", "errors", "WREF"],
         join(testdir, "1.2", "errors", "WREF.dml"),
         api_version="5"))
+    all_tests.append(CTestCase(
+        ["1.4", "errors", "WLOGMIXUP"],
+        join(testdir, "1.4", "errors", "WLOGMIXUP.dml"),
+        dmlc_extraargs = ["--warn=WLOGMIXUP"]))
 
 if get_simics_major() == "7":
     all_tests.append(CTestCase(
-        ["1.4", "errors", "T_ETYPE_integer_t"],
+        ["1.4", "errors", "ETYPE_integer_t"],
         join(testdir, "1.4", "errors", "ETYPE_integer_t.dml"),
         api_version="7"))
+    all_tests.append(CTestCase(
+        ["1.4", "errors", "WLOGMIXUP"],
+        join(testdir, "1.4", "errors", "WLOGMIXUP.dml"),
+        api_version="7"))
```

Comment:
This is fine, but one alternative to consider is `/// DMLC-FLAG --api-version=7` in the test. Deserves an explanatory comment too ("only enabled by default in 7 and newer"): in a couple of years, api=7 will indicate it tests some legacy behaviour.

Reply:
`--api-version=7` won't be accepted in simics-base/6, so that's not an option.